backend/src/data/repositories/refreshTokenRepository.ts

```diff
@@ -0,0 +1,23 @@
+import { EntityRepository, AbstractRepository } from 'typeorm';
+import { IRefreshToken } from '../../common/models/IRefreshToken';
+import { RefreshToken } from '../entities/RefreshToken';
+
+@EntityRepository(RefreshToken)
+class RefreshTokenRepository extends AbstractRepository<RefreshToken> {
```

Comment:
https://github.com/BinaryStudioAcademy/bsa-2020-chatito/blob/dev/backend/src/data/repositories/userRepository.ts

Reply:
please check how another repo is implemented